config/locales/en.yml

```
@@ -1999,6 +2003,10 @@ en:
           title1: You're done! Let's review before you submit your state tax return.
           your_refund: Your estimated refund
           your_tax_owed: Your estimated tax owed
+          taxes_owed_html: |
+            <b>Your estimated refund or taxes owed</b> $0. <br/>
```

I think we prefer <strong> tags over <b> tags for accessibility reasons 
```suggestion
            <strong>Your estimated refund or taxes owed</strong> $0. <br/>
```

---

Done